utils/products.js

```diff
@@ -1,152 +1,664 @@
+      "https://github.com/ds-lcapellino",
+      "https://github.com/mkanal",
     ],
     mailTo: "tractusx-dev@eclipse.org?subject=Request Trace-X Team",
     hasBoard: false,
-    showVersion: true
+    showVersion: true,
+  },
+  {
+    productName: "tractus-x-release",
+    productDescription: "TBD",
+    githubRepo: ["https://github.com/eclipse-tractusx/tractus-x-release"],
+    committers: [
+      "https://github.com/tom-rm-meyer-ISST",
+      "https://github.com/arnoweiss",
+      "https://github.com/stephanbcbauer",
+      "https://github.com/evegufy",
+    ],
+    mailTo: "tractusx-dev@eclipse.org?subject=Request tractus-x-release Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "tractus-x-umbrella",
+    productDescription: "TBD",
+    githubRepo: ["https://github.com/eclipse-tractusx/tractus-x-umbrella"],
+    committers: [
+      "https://github.com/ds-jhartmann",
+      "https://github.com/almadigabor",
+      "https://github.com/evegufy",
+      "https://github.com/jzbmw",
+      "https://github.com/tunacicek",
+    ],
+    mailTo: "tractusx-dev@eclipse.org?subject=Request tractus-x-umbrella Team",
+    hasBoard: false,
+    showVersion: true,
   },
 
   {
     productName: "Tractus-X EDC",
-    productDescription: "The project provides the pre-built control- and data-plane docker images and helm charts of the Eclipse Dataspace Connector Project.",
-     githubRepo: [
-      "https://github.com/eclipse-tractusx/tractusx-edc"
+    productDescription:
+      "The project provides the pre-built control- and data-plane docker images and helm charts of the Eclipse Dataspace Connector Project.",
+    githubRepo: [
+      "https://github.com/eclipse-tractusx/tractusx-edc",
+      "https://github.com/eclipse-tractusx/tractusx-edc-template",
     ],
     committers: [
       "https://github.com/paullatzelsperger",
-      "https://github.com/florianrusch-zf"
+      "https://github.com/wolf4ood",
+      "https://github.com/ndr-brt",
+      "https://github.com/tuncaytunc-zf",
     ],
     mailTo: "tractusx-dev@eclipse.org?subject=Request Tractus-X EDC Team",
     hasBoard: true,
-    showVersion: true
+    showVersion: true,
   },
-
   {
-    productName: "Policy Hub",
-    productDescription: "The Policy Hub enables data providers, consumers and app providers to access a single-point-of-truth for Catena-X policies, attributes and templates for policy rules.",
-     githubRepo: [
-      "https://github.com/eclipse-tractusx/policy-hub"
+    productName: "tractusx-profiles",
+    productDescription: "TBD",
+    githubRepo: ["https://github.com/eclipse-tractusx/tractusx-profiles"],
+    committers: [
+      "https://github.com/jimmarino",
+      "https://github.com/arnoweiss",
+      "https://github.com/wolf4ood",
     ],
+    mailTo: "tractusx-dev@eclipse.org?subject=Request tractusx-profiles Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "tractusx-quality-checks",
```

Comment:
Has this list been generated by auomation? If so, I would consider adding the `--no-archive` flag. 
This is only an enabling automation repo and not a product. It is also archived.
If you want to keep it anyways, please remove me from the list of committers